install_playground.sh

```diff
@@ -3,7 +3,36 @@ cd interface
 yarn install
 
 # install and run backend
-cd ../backend && python3 -m venv venv
-source ./venv/bin/activate
-pip install wheel
-pip install -r requirements.txt
\ No newline at end of file
+cd ../backend
+# Create a conda environment (to install Diart requirements) with Python 3.8 and activate it
+echo "Current shell: $SHELL"
+
+conda create -n env python=3.8
```

Comment:
Why anchor to 3.8?

Reply:
Diart anchors to 3.8 in its installation guide and it's not stated anywhere that it supports later versions, I'll have to do some testing of my own to see if I can get it to work in Python 3.8
This is also limiting when wanting to implement Whisper implementations like WhisperX

Reply:
If it all works, I don't think it's a priority at the moment. No need to research and upgrade for now.